test/decorator-request.test.js

```diff
@@ -0,0 +1,24 @@
+import { test } from 'tap'
+import Fastify from 'fastify'
+
+import slowDownPlugin from '../index.js'
+import { DEFAULT_OPTIONS } from '../lib/constants.js'
+
+test('should contain the slowDown request decorator', async t => {
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin)
+  t.teardown(() => fastify.close())
+
+  fastify.get('/', async (req, reply) => reply.send(req.slowDown))
```

Comment:
can you please use idiomatic fastify conventions? if the handler is async, you don't need to use reply.send